assets/js/src/form_editor/blocks/custom_date/custom_field_settings.jsx

```diff
@@ -51,6 +51,7 @@ const CustomFieldSettings = ({
     return (
       <SelectControl
         label={MailPoet.I18n.t('customFieldDateFormat')}
+        data-automation-id="settings_custom_date_order"
```

Comment:
Label and value have suffix _format_. Why did you use suffix _order_ here?

Reply:
Yes, you're right. My fault.